pkg/pod/pod.go

```diff
@@ -384,3 +384,13 @@ func shouldAddReadyAnnotationOnPodCreate(ctx context.Context, sidecars []v1beta1
 	cfg := config.FromContextOrDefaults(ctx)
 	return !cfg.FeatureFlags.RunningInEnvWithInjectedSidecars
 }
+
+// WillOverwritePodSetAffinity returns a bool indicating whether the
+// affinity for pods will be overwritten with affinity assistant.
+func WillOverwritePodSetAffinity(taskRun *v1beta1.TaskRun) bool {
```

Comment:
Is it possible to declare this in a package so that it can be _private_?

Reply:
Thank you for the review. I think this is related to the [comment below](https://github.com/tektoncd/pipeline/pull/2859#discussion_r445822195), i.e. if the `WillOverwritePodSetAffinity` is moved to the `reconciler` package, then it can be made private.